### PR TITLE
refactor: Linkedin Patch Updates

### PR DESCRIPTION
**Pull request by lucas5thorneRyanMiranda**
Adds the following patches to the LinkedIn connector:

1. Adds the approximateUniqueImpressions column with data to adAnalytics (Replaces unnecessary pivot column to account for 80 column request limit)
2. Updates the Default LinkedIn-Version to 202211 (To support approximateUniqueImpressions)
3. Fixes issue with get_next_page_token cutting off a row of data in accountUser
4. Data pulling properly in adTargetingFacets column

### CODE REVIEW

File: tap_linkedin_ads/streams.py [hunk 1]
```diff
@@ -873,30 +873,29 @@ class Campaigns(LinkedInAdsStream):
...
+                                            "urn:li:adTargetingFacet",
+                                            th.ArrayType(
+                                                Property("urn:li:geo", StringType),
+                                            ),
+                                        ),
+                                        Property(
+                                            "urn:li:adTargetingFacet",
+                                            th.ArrayType(
+                                                Property(
+                                                    "urn:li:adSlotSize", StringType
```
Comment (edgarrmondragon) [inline tap_linkedin_ads/streams.py:896]:
```suggestion
                                                    "urn:li:adSlotSize",
                                                    StringType,
```

**Review by edgarrmondragon — APPROVED** [review]

File: tap_linkedin_ads/streams.py [hunk 2]
```diff
@@ -282,7 +282,7 @@ def adanalyticscolumns(self) -> list[str]:
...
             "adUnitClicks,videoThirdQuartileCompletions,cardClicks,likes,viralComments,viralVideoMidpointCompletions,viralVideoThirdQuartileCompletions,oneClickLeads,fullScreenPlays,viralCardImpressions,follows,videoStarts,videoFirstQuartileCompletions,textUrlClicks,pivotValue,reactions,viralReactions,externalWebsitePostClickConversions,viralOtherEngagements,costInLocalCurrency",
-            "viralVideoStarts,viralRegistrations,viralJobApplyClicks,viralJobApplications,jobApplications,jobApplyClicks,viralExternalWebsiteConversions,postViewRegistrations,companyPageClicks,documentCompletions,documentFirstQuartileCompletions,documentMidpointCompletions,documentThirdQuartileCompletions,downloadClicks,viralDocumentCompletions,viralDocumentFirstQuartileCompletions,viralDocumentMidpointCompletions,viralDocumentThirdQuartileCompletions,viralDownloadClicks,impressions",
```
Comment (pnadolny13) [inline tap_linkedin_ads/streams.py:285]: @lucas5thorneRyanMiranda are these meaning to remove the `viralDocumentThirdQuartileCompletions` column? I didnt see anything noted in the PR description and it looks like that column is still referenced in the schema https://github.com/MeltanoLabs/tap-linkedin-ads/pull/63/files#diff-d3caa667c1dc7e3ada59d3189681817406a6dba0c45df80d33407bcb0ed05dc3L251.

Comment (NeilGorman104) [reply tap_linkedin_ads/streams.py:285]: Hey Pat,

Good catch. This column was replaced by approximateUniqueImpressions due to the limit of fields we can pull from, but was only meant as a stand-in to test that we are pulling approximateUniqueImpressions. We added this column back and actually removed the pivot column, which is not a part of the fivetran table and is a static value (“Campaign” or “Creative”) every time.

Comment (pnadolny13) [reply tap_linkedin_ads/streams.py:285]: @NeilGorman104 that makes sense. Thanks for sharing that context, very helpful!

**Review by pnadolny13 — APPROVED** [review]

**Pull request merged**